pkg/tests/api/alerting/api_admin_configuration_test.go

```diff
@@ -25,7 +25,11 @@ import (
 	"github.com/grafana/grafana/pkg/tests/testinfra"
 )
 
-func TestAdminConfiguration_SendingToExternalAlertmanagers(t *testing.T) {
+func TestIntegrationAdminConfiguration_SendingToExternalAlertmanagers(t *testing.T) {
```

Comment:
If we rename them including the `Integration` they will run against [MySQL integration tests](https://github.com/grafana/grafana/blob/5bb99775bb995436e9fcf723ed1e622b685a9f1f/.drone.yml#L1725) (same for PostgreSQL integration tests); I'm not sure if we want that.

Reply:
Oh good catch :+1: It seems like it added like 5-6 min to the integration tests duration. For now I could skip them if not sqlite is used.

Reply:
Pushed a suggestion to handle this. Not sure it make sense, WDYT?